src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -36,29 +36,12 @@
 */
 public class ADDR_Addresses_TDTM extends TDTM_Runnable {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.